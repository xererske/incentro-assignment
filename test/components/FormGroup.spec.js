import { mount } from "@vue/test-utils"
import FormGroup from "~/components/form-group.vue"
import { commonRendersCorrectly } from "~/test/common/RendersCorrectly"

const factory = () => {
  return mount(FormGroup, {
    propsData: {
      classes: "justify-center",
    },
  })
}

commonRendersCorrectly("FormGroup", factory)
