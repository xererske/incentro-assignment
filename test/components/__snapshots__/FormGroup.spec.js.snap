// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly: FormGroup Renders correctly 1`] = `<div class="flex flex-row mb-3 justify-center"></div>`;
